controllers/usersignup/usersignup_controller.go

```diff
@@ -798,6 +804,16 @@ func (r *Reconciler) provisionMasterUserRecord(logger logr.Logger, config toolch
 	// increment the counter of MasterUserRecords
 	domain := metrics.GetEmailDomain(mur)
 	counter.IncrementMasterUserRecordCount(logger, domain)
+
+	//check MUR was actually created
+	murRetrieved := &toolchainv1alpha1.MasterUserRecord{}
+	err = r.Client.Get(context.TODO(), types.NamespacedName{Name: mur.Name, Namespace: mur.Namespace}, murRetrieved)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			return fmt.Errorf(`MUR %s created isn't available yet`, mur.Name)
+		}
+		return err
+	}
```

Comment:
I think we can remove this check because we're creating the MUR just above here so I believe checking if it exists here is redundant because we should get an error if the creation failed.

Reply:
done in [d6bfc32](https://github.com/codeready-toolchain/host-operator/pull/670/commits/d6bfc3296a49c51e700d78063c0e676e719079a5)